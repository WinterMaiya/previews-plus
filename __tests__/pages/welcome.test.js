import { render, screen, waitFor } from "@testing-library/react";
import "@testing-library/jest-dom";
import WelcomePage from "../../pages/welcome";

describe("Welcome Page", () => {
	// Finish test when welcome page is created
	it("Snapshot", async () => {
		const { container } = render(<WelcomePage />);
		expect(container).toMatchSnapshot();
	});
});
