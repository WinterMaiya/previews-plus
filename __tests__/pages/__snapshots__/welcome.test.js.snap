// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Page Snapshot 1`] = `
<div>
  <div>
    <section
      class="text-white"
    >
      <div
        class="container"
      >
        <h1
          class="text-center text-primary satisfy-font bg-dark mt-4 make-logo-bigger"
        >
          Previews+
        </h1>
        <p
          class="text-center fst-italic fw-light"
        >
          Watch 
          <strong>
            unlimited
          </strong>
           previews
        </p>
      </div>
      <div
        class="parallax bg-1"
      />
      <div
        class="container my-3"
      >
        <div
          class="row text-center"
        />
        <div
          class="row text-center "
        >
          <div>
            <p
              class="fs-5"
            >
              <strong
                class="fst-italic"
              >
                Previews+
              </strong>
               lets you stream your favorite movies and tv shows'
               
              <strong
                class="text-primary"
              >
                trailers!
              </strong>
            </p>
            <p
              class="fs-6"
            >
              Create an account through
               
              <strong
                class="text-primary"
              >
                Github
              </strong>
               or
               
              <strong
                class="text-primary"
              >
                Google
              </strong>
              , make your own watch profile, and start browsing our list of the most popular (or obscure) trailers on the web:
            </p>
          </div>
          <div
            class="col"
          >
            <button
              class="text-white btn btn-outline-primary"
            >
              Start Here
            </button>
          </div>
        </div>
      </div>
      <div
        class="parallax bg-2"
      />
      <div
        class="container my-5"
      >
        <div
          class="row text-center"
        >
          <div
            class="col"
          >
            <p
              class="fs-5"
            >
              Add videos you want to see to your
               
              <strong
                class="text-primary"
              >
                Watchlist 
              </strong>
              or rate videos you've seen before. Our algorithm learns your preferences in order to show you
              <strong
                class="text-primary"
              >
                 new 
              </strong>
              shows and movies you will enjoy!
            </p>
          </div>
        </div>
      </div>
      <div
        class="parallax bg-3"
      />
      <div
        class="container my-5"
      >
        <div
          class="row text-center"
        >
          <div
            class="col"
          >
            <h3
              class="fs-4"
            >
              About the 
              <strong
                class="text-primary"
              >
                Project
              </strong>
            </h3>
            <p>
              This project was created by
               
              <a
                class="fst-italic text-white"
                href="https://www.linkedin.com/in/maiya-winter/"
              >
                Maiya Winter
              </a>
               using
               
              <strong>
                Next.js, Prisma, PostgreSQL, and Next-Auth
              </strong>
              . You can view this project on
               
              <a
                class="fst-italic text-white"
                href="https://github.com/WinterMaiya/previews-plus"
              >
                Github
              </a>
              . I created this project as an attempt to replicate popular streaming services. This was my first project with Next.js so it was an excellent adn exciting learning experience for me to learn and improve. Any advice or questions are more then welcome. You can contact me through my 
              <strong>
                Github 
              </strong>
              page.
            </p>
          </div>
        </div>
      </div>
      <div
        class="parallax bg-4"
      />
      <div
        class="container my-3"
      >
        <div
          class="row text-center"
        >
          <div
            class="col"
          >
            <h3
              class="display-6 my-2"
            >
              Ready to get started?
            </h3>
            <button
              class="text-white btn btn-outline-primary my-3"
            >
              Start Here
            </button>
          </div>
        </div>
      </div>
      <div
        class="parallax bg-5"
      />
      <div
        class="container my-3"
      >
        <div
          class="row text-center"
        >
          <div
            class="col"
          >
            <p>
              External API data thanks to 
              <strong>
                TMDB:
              </strong>
            </p>
            <div>
              <a
                href="https://www.themoviedb.org"
              >
                <img
                  height="100"
                  src="/images/tmdb_logo.svg"
                  width="100"
                />
              </a>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>
</div>
`;
